src/command/upload.ts

```diff
@@ -0,0 +1,134 @@
+import { Option, LeafCommand, Argument } from 'furious-commander'
+import { RootCommand } from './root-command'
+import * as FS from 'fs'
+import * as Path from 'path'
+import { sleep } from '../utils'
+import { Tag } from '@ethersphere/bee-js/dist/types'
+import { SingleBar, Presets } from 'cli-progress'
+import { bold, green, dim, red } from 'kleur'
+
+export class Upload extends RootCommand implements LeafCommand {
+  // CLI FIELDS
+
+  public readonly name = 'upload'
+
+  public readonly description = 'Upload file to Swarm'
+
+  @Argument({ key: 'path', describe: 'Path of the file (even directory)', demandOption: true })
```

Comment:
yeah, I tried to keep argument params the same as option params, and `yargs` use `demandOption` boolean for option be required. I will change both property to `required` in `furious-commander`.

Reply:
In that case I think it would be better to use `Argument` for arguments that are required and `OptionalArgument` for arguments that are not required and not expose the `demandOption` flag.

Reply:
Or as @nugaon suggested, use `required` keyword